### PR TITLE
added platform support

### PR DESCRIPTION
**Pull request by zween27**
distinguish between Windows and other platforms for call of readelf

### CODE REVIEW

File: WCS.py [hunk 1]
```diff
@@ -12,7 +13,10 @@
 su_ext = '.su'
 obj_ext = '.o'
 manual_ext = '.msu'
-read_elf_path = "arm-none-eabi-readelf.exe" # You may need to enter the full path here
+if platform.system() == 'Windows':
+    read_elf_path = "arm-none-eabi-readelf.exe" # You may need to enter the full path here
```
Comment (saloid) [inline WCS.py:17]: `arm-none-eabi-readelf` would work in windows too (without .exe extension)

**Comment by DEvil0000** [comment]
in case someone is looking for this (on windows + ".exe" I guess)
```
//if arduino
read_elf_path = "avr-readelf"
```

**Comment by ericLemanissier** [comment]
@zween27 
I added the possibility to pass the readelf executable name as an argument on the command line in https://github.com/PeterMcKinnis/WorstCaseStack/commit/e357e3e2218b3d39f179366f90411470c0e3dc5b
this should fix the platform issue in a more general way. If it is ok for you, feel free to close this PR

**Comment by PeterMcKinnis** [comment]
Closing this as it was addressed by:

> I added the possibility to pass the readelf executable name as an argument on the command line in [e357e3e](https://github.com/PeterMcKinnis/WorstCaseStack/commit/e357e3e2218b3d39f179366f90411470c0e3dc5b)
> this should fix the platform issue in a more general way. If it is ok for you, feel free to close this PR

**Pull request closed without merging**